bin/verilator

```diff
@@ -2928,7 +3059,8 @@ per the C standard (it's unspecified in Verilog).
 
 Specifies the entire begin/end block should be ignored for coverage
 analysis.  Must be inside a basic block, e.g. within a begin/end pair.
-Same as /* verilator coverage_block_off */.
+Same as /* verilator coverage_block_off */ and coverage_block_off in
+configuration files.
 
```

Comment:
"... and C<coverage_block_off> in ....

(So the font makes it clear this is a command. Ditto others.)